content/en/blog/_posts/2022-10-03-current-state-2019-third-party-audit.md

```diff
@@ -0,0 +1,116 @@
+| 20     | Adding credentials to containers by default is unsafe                       | [#81128](https://github.com/kubernetes/kubernetes/issues/81128) | Closed, fixed by [#89193](https://github.com/kubernetes/kubernetes/pull/89193)                                                                                                                                           |
+| 21     | kubelet liveness probes can be used to enumerate host network               | [#81129](https://github.com/kubernetes/kubernetes/issues/81129) | Needs a KEP                                                                                                                                                                                                                     |
+| 22     | iSCSI volume storage cleartext secrets in logs                              | [#81130](https://github.com/kubernetes/kubernetes/issues/81130) | closed, fixed by [#81215](https://github.com/kubernetes/kubernetes/pull/81215)                                                                                                                                                  |
+| 23     | Hard coded credential paths                                                 | [#81131](https://github.com/kubernetes/kubernetes/issues/81131) | closed, awaiting more evidence                                                                                                                                                                                                  |
+| 24     | Log rotation is not atomic                                                  | [#81132](https://github.com/kubernetes/kubernetes/issues/81132) | Fix needed                                                                                                                                                                                                                      |
+| 25     | Arbitrary file paths without bounding                                       | [#81133](https://github.com/kubernetes/kubernetes/issues/81133) | Fix needed.                                                                                                                                                                                                                     |
+| 26     | Unsafe JSON construction                                                    | [#81134](https://github.com/kubernetes/kubernetes/issues/81134) | Partially fixed                                                                                                                                                                                                                 |
```

Comment:
are the details on this in the actual issue? maybe we can link to a specific comment about the partial fix?

Reply:
@cji I am not confident a specific comment in the issue explains the current state accurately. So the default of commend on the issue to show up to help would still apply here.

Reply:
makes sense - thank you!